lib/cache.js

```diff
@@ -1,8 +1,31 @@
 'use strict'
 
+const Winston = require('winston')
 const LevelUp = require('level')
+const Sublevel = require('level-sublevel')
 const Path = require('path')
 
-const path = Path.join(__dirname, '..', 'data')
+const path = Path.join(__dirname, '..', 'cache')
 
-module.exports = LevelUp(path)
+const leveldb = LevelUp(path)
+// main cache DB path
+const db = Sublevel(leveldb)
+
+exports.addSubLevelCacheDB = (levelKey) => {
```

Comment:
Main reason was to keep all leveldb code in a single file/service, but can be done directly where it is used, Would we then still need the cache.js file? it would then only create the main db level